### PR TITLE
Add notes on JWT authentication

### PR DESCRIPTION
**Pull request by mbeerman**
Description
----
In preparation for the rollout of Bulk Data, add notes on JWT authentication.

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

Screen Shot 2022-09-29 at 7 42 17 PM

### CODE REVIEW

**Review by dclippard — APPROVED** [review]
Looks good

File: content/authorization.md [hunk 1]
```diff
@@ -1037,9 +1043,11 @@ curl -X POST 'https://authorization.cerner.com/tenants/ec2458f2-1e24-41c8-b71b-0
   -d 'grant_type=client_credentials&scope=system%2FObservation.read%20system%2FPatient.read'
 </pre>
 
-_Note_:  This access model is only supported on resources
-where explicitly indicated in their respective
-FHIR<sup>®</sup> API resource documentation.
+#### JWT Authentication ####
+
+Alternatively, clients may authenticate themselves using signed JSON Web Tokens (JWTs). This process is similar to the above, but uses ``Authorization: Bearer`` instead. This is the preferred mode of authentication for Bulk Data clients. For complete details, see [Java Web Token (JWT)](https://www.rfc-editor.org/rfc/rfc7519) and [JSON Web Token (JWT) Profile for OAuth 2.0 Client Authentication and Authorization Grants](https://www.rfc-editor.org/rfc/rfc7523).
```
Comment (SA030800) [inline content/authorization.md:1048]: Note sure if we avoid linking to FHIR spec from this page if not would it be better to make the phrase  'Bulk Data clients' a link that links to this page: http://www.hl7.org/fhir/smart-app-launch/backend-services.html?

Comment (mbeerman) [reply content/authorization.md]: Sure, added. Better now?

Comment (SA030800) [reply content/authorization.md]: yeah, even better they can get to the page I have from the link you have

**Review by SA030800 — APPROVED** [review]

**Comment by ebbp123** [comment]
Post-Deployment Validation
----
Screen Shot 2022-09-30 at 9 42 57 AM
Screen Shot 2022-09-30 at 9 42 38 AM

**Pull request merged**